lib/protocol/SqlString.js

```diff
@@ -66,16 +66,16 @@ SqlString.arrayToList = function(array, timeZone) {
 
 SqlString.format = function(sql, values, stringifyObjects, timeZone) {
   values = values == null ? [] : [].concat(values);
```

Comment:
@ronkorving feel free to make a PR :) And remember that the `values` argument may not always be an array.